### PR TITLE
Traitor PDA and gear enhancements. **Feedback Needed**

### PR DESCRIPTION
**Pull request by Sparrowbird611**
This one is a bit big, so forgive me if I've forgotten to mention anything.

First off, adds two new sections, "Chemicals" and "Armor and tactical wear." The first is filled with a multitude of items, primarily poisons, but also some smaller helping items, and a 1 TC drug for drug dealing gimmicks. The second holds a few new additions to armor, including the additions of the specialized armor types, and a new chameleon armor set.

Next off, tweaks the prices on a few items. The combat medical kit was lowered to 16 TC, and the stabilization medical kit to 6. The syndicate voidsuit is now 22 TC. The cane sword was raised to 10 TC. Sleepy pen has had it's price raised to 18. Chameleon kit lowered to 10 TC.

Speaking of the cane sword, it's an actual sword now, holding a steel katana inside of it. Because cane knives are lame.

The heavy armor now has a siemon_Coefficient, helping to prevent the user from being stunlocked by shock beams. This seems like an oversight, given the helmet had it, but the body armor did not.

Along with the already mentioned new additions, here's a few more

Camera flash: An advanced flashbulb in the form of a camera. 10 TC
Energy shield: A high tech take on an old classic, blocking both ranged and melee weaponry. 22 TC.
Switchblade: Because we all wanna be gangsters. 4 TC
Nabber insulated gloves: Because some of us aren't blessed with five fingers. 2 TC
Tacticool gear: Ranges from shoulder holsters to tactical belts and combat gloves. Prices are low all around.

**Feedback**

I need thoughts on the chemical section prices primarily. Tell me if any of the prices look too high/low, and if there's any decent chemicals I should add, though I would prefer to abstain from adding any more poisons, as half this section is already filled with them.

### CODE REVIEW

**Comment by EldritchSigma** [comment]
Siemens coefficient not simon aaa

**Comment by Sparrowbird611** [comment]
I said siemen coeff though. I also don't know why it's saying the medical files are conflicting, the only thing I touched were the prices. I'll look at it when I get the chance.

**Comment by Roland410** [comment]
Because bone juice was merged afaik.

**Comment by Sparrowbird611** [comment]
Ah, I see now.

Guess I can just copy the code from that PR on to this one to fix it.

File: code/datums/uplink/armor and tactical wear.dm [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	name = "Riot Kit"
+	desc = "Contains riot arm guards, riot leg guards, and a riot helmet."
+	item_cost = 2
+	path = /obj/item/weapon/storage/box/syndie_kit/riotkit
+
+/datum/uplink_item/item/armor/syndiehardsuit
+	name = "Chameleon Hardsuit"
+	desc = "An advanced hardsuit with built in armor and tactical chameleon plating that enables it to alter it's appearance."
+	item_cost = 30
+
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:115]: Cool, but might want to add an item here.

File: code/datums/uplink/armor and tactical wear.dm [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/mask/balaclava
+
+/datum/uplink_item/item/armor/bandolier
+	name = "Bandolier"
+	desc = "Pack it full of ammo and go wild."
+	item_cost = 1
+	path = /obj/item/clothing/accessory/storage/bandolier
+
+/datum/uplink_item/item/armor/beltholster
+	name = "Belt Holster"
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:40]: Just name this security holster belt.

File: code/datums/uplink/armor and tactical wear.dm [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+/datum/uplink_item/item/armor/holster
+	name = "Shoulder Holster"
+	desc = "A holster that attached to the shoulder."
+	item_cost = 1
+	path = /obj/item/clothing/accessory/storage/holster
+
+/datum/uplink_item/item/armor/plate_carrier
+	name = "Plate Carrier"
+	desc = "A modular plate carrier. It comes unequipped, but can be upgraded with various forms of plate."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:54]: Should probably be more expensive even if it's empty.

File: code/datums/uplink/armor and tactical wear.dm [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/suit/armor/pcarrier
+
+/datum/uplink_item/item/armor/plate_carrierriot
+	name = "Riot Carrier"
+	desc = "A modular plate carrier. It comes equipped with a riot plate."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:60]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/suit/armor/riot
+
+/datum/uplink_item/item/armor/plate_carrierablative
+	name = "Ablative Carrier"
+	desc = "A modular plate carrier. It comes equipped with an ablative plate."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:66]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/suit/armor/laserproof
+
+/datum/uplink_item/item/armor/plate_carrierballistic
+	name = "Ballistic Carrier"
+	desc = "A modular plate carrier. Comes equipped with a ballistic plate."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:72]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 7]
```diff
@@ -0,0 +1,132 @@
+/************************
+* Armor & Tactical Wear *
+************************/
+/datum/uplink_item/item/armor
+	category = /datum/uplink_category/armor
+
+/datum/uplink_item/item/armor/heavy_armor
+	name = "Heavy Armor Vest and Helmet"
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:8]: I'm fairly sure this is an item already.

File: code/datums/uplink/armor and tactical wear.dm [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+/datum/uplink_item/item/armor/plate_carrierballistic
+	name = "Ballistic Carrier"
+	desc = "A modular plate carrier. Comes equipped with a ballistic plate."
+	item_cost = 1
+	path = /obj/item/clothing/suit/armor/bulletproof
+
+/datum/uplink_item/item/armor/platepouch
+	name = "Plate Pouch"
+	desc = "A pouch that can attach to a plate carrier. Holds two items."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:78]: I hope this is WIP. Also pointless when you have the 4 slot one.

File: code/datums/uplink/armor and tactical wear.dm [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/accessory/storage/pouches/grey
+
+/datum/uplink_item/item/armor/platepouchlarge
+	name = "Large Plate Pouch"
+	desc = "A large pouch that can attach to a plate carrier. Holds four items, and slows you down more than a regular pouch."
+	item_cost = 1
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:84]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/clothing/accessory/storage/pouches/large/grey
+
+/datum/uplink_item/item/armor/mediumkit
+	name = "Medium Plate Kit"
+	desc = "Contains a medium chest plate, medium arm guards, medium leg guards, and a medium helmet."
+	item_cost = 2
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:90]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/weapon/storage/box/syndie_kit/mediumkit
+
+/datum/uplink_item/item/armor/ablativekit
+	name = "Ablative Kit"
+	desc = "Contains ablative arm guards, ablative leg guards, and an ablative helmet."
+	item_cost = 2
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:96]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/weapon/storage/box/syndie_kit/ablativekit
+
+/datum/uplink_item/item/armor/ballistickit
+	name = "Ballistic Kit"
+	desc = "Contains ballistic arm guards, riot leg guards, and a ballistic helmet."
+	item_cost = 2
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:102]: I hope this is WIP, also description says riot leg guards?

File: code/datums/uplink/armor and tactical wear.dm [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+	path = /obj/item/weapon/storage/box/syndie_kit/ballistickit
+
+/datum/uplink_item/item/armor/riotkit
+	name = "Riot Kit"
+	desc = "Contains riot arm guards, riot leg guards, and a riot helmet."
+	item_cost = 2
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:108]: I hope this is WIP.

File: code/datums/uplink/armor and tactical wear.dm [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+/datum/uplink_item/item/armor/stealtharmor
+	name = "Chameleon Armor"
+	desc = "A two piece set of chameleon armor that covers the entire body. Due to space and technical limitations, it is only effective against handgun tier weaponry."
+	item_cost = 12
+	path = /obj/item/weapon/storage/backpack/chameleonarmor/sydie_kit
+
+/datum/uplink_item/item/armor/combatgloves
+	name = "Insulated Combat Gloves"
+	desc = "A pair of gloves that give moderate hand protection."
+	item_cost = 2
```
Comment (Roland410) [inline code/datums/uplink/armor and tactical wear.dm:125]: Probably should be like 6 TC.

File: code/datums/uplink/chemicals.dm [hunk 15]
```diff
@@ -0,0 +1,95 @@
...
+	path = /obj/item/weapon/reagent_containers/glass/bottle/drugmix
+
+/datum/uplink_item/item/chemicals/hfp
+	name = "Vial of Heptafluoropropane"
+	desc = "One of the most dangerous chemicals you could think of, easily vialed up. Be extra careful not to drink it!"
+	item_cost = 64
+	path = /obj/item/weapon/reagent_containers/glass/beaker/vial/hfp
+
+/datum/uplink_item/item/chemicals/chloral
+	name = "Chloral Hydrate"
```
Comment (Roland410) [inline code/datums/uplink/chemicals.dm:44]: If you add Chloral, might as well just change the pen back to have it imo.

File: code/datums/uplink/chemicals.dm [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+	path = /obj/item/weapon/reagent_containers/glass/bottle/lipozine
+
+/datum/uplink_item/item/chemicals/acid
+	name = "Polytrinic Acid"
+	desc = "A powerful acid, capable of causing severe physical harm to others."
+	item_cost = 8
+	path = /obj/item/weapon/reagent_containers/glass/bottle/pacid
+
+/*
+/datum/uplink_item/item/chemicals/threeeye
```
Comment (Roland410) [inline code/datums/uplink/chemicals.dm:92]: Don't leave commented-out code, and as we spoke, I have plans for this.

File: code/datums/uplink/chemicals.dm [hunk 17]
```diff
@@ -0,0 +1,95 @@
...
+/datum/uplink_item/item/chemicals/lipozine
+	name = "Lipozine"
+	desc = "A weight loss chemical, which causes the user to become hungry. A surprisingly effective poison against Unathi."
+	item_cost = 6
+	path = /obj/item/weapon/reagent_containers/glass/bottle/lipozine
+
+/datum/uplink_item/item/chemicals/acid
+	name = "Polytrinic Acid"
+	desc = "A powerful acid, capable of causing severe physical harm to others."
+	item_cost = 8
```
Comment (Roland410) [inline code/datums/uplink/chemicals.dm:88]: Pacid is quite strong to only be 8 TC, at least 20-40 I'd say.

File: code/datums/uplink/devices and tools.dm [hunk 18]
```diff
@@ -163,3 +163,9 @@
 	path = /obj/effect/spawner/newbomb/traitor
 	desc = "A remote-activated phoron-oxygen bomb assembly with an included signaler. \
 			A flashing disclaimer begins with the warning 'SOME DISASSEMBLY/REASSEMBLY REQUIRED.'"
+
+/datum/uplink_item/item/tools/nabbergloves
+	name = "Insulated Serpentid Gloves"
+	desc = "Insulated gloves, for Serpentids. Need I say more?"
+	item_cost = 2
+	path = /obj/item/clothing/gloves/nabber
```
Comment (Roland410) [inline code/datums/uplink/devices and tools.dm:171]: GAS can't really be traitors still, so this is quite useless.

Comment (Sparrowbird611) [reply code/datums/uplink/devices and tools.dm:171]: GAS can be traitors, I do it myself a fair bit. I'm leaving this.

File: code/datums/uplink/chemicals.dm (hunk 16 above)
Comment (Sparrowbird611) [reply code/datums/uplink/chemicals.dm:92]: I'll remove this, as I forgot it.

File: code/datums/uplink/chemicals.dm (hunk 15 above)
Comment (Sparrowbird611) [reply code/datums/uplink/chemicals.dm:44]: Honestly considering making it so the pen starts empty, and you have to fill it. Dunno, we'll see.  Either way, out of the bounds of this PR.

File: code/datums/uplink/armor and tactical wear.dm (hunk 14 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:125]: This was an oversight, as these are actually regular combat gloves, not insulated ones. I have to rename these.

File: code/datums/uplink/armor and tactical wear.dm (hunk 3 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:54]: Eh, you can acquire these in the loadout quite easily. I may considering raising it to 2 TC.

File: code/datums/uplink/armor and tactical wear.dm (hunk 2 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:40]: Alright, I'll get to that.

File: code/datums/uplink/armor and tactical wear.dm (hunk 1 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:115]: Oversight, I forgot to remove this item. It's a potential addition, but outside what I planned for this PR.

File: code/datums/uplink/chemicals.dm (hunk 17 above)
Comment (Sparrowbird611) [reply code/datums/uplink/chemicals.dm:88]: It's also all over maintenance. I will raise it to 12-16 however.

File: code/datums/uplink/armor and tactical wear.dm (hunk 4 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:60]: Specialist armor can easily be bypassed by swapping weapons, and security carries an incredibly diverse loadout. I will consider increasing the price a bit however.

This goes for the bottom review, and other similar ones.

File: code/datums/uplink/armor and tactical wear.dm (hunk 3 above)
Comment (Roland410) [reply code/datums/uplink/armor and tactical wear.dm:54]: Only armed roles can get it though, not everyone. Only the civilian grey is available.

File: code/datums/uplink/chemicals.dm (hunk 17 above)
Comment (Roland410) [reply code/datums/uplink/chemicals.dm:88]: That's sulphuric you're thinking of. Polyacid is way stronger.

Comment (Sparrowbird611) [reply code/datums/uplink/chemicals.dm:88]: raised to 14

File: code/datums/uplink/chemicals.dm (hunk 16 above)
Comment (Sparrowbird611) [reply code/datums/uplink/chemicals.dm:92]: removed

File: code/datums/uplink/armor and tactical wear.dm (hunk 14 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:125]: Renamed

File: code/datums/uplink/armor and tactical wear.dm (hunk 4 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:60]: Price raised to 2 becuase melee is shit here, other specialist forms raised as well.

File: code/datums/uplink/armor and tactical wear.dm (hunk 5 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:66]: Same as above.

File: code/datums/uplink/armor and tactical wear.dm (hunk 3 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:54]: Raised

File: code/datums/uplink/armor and tactical wear.dm (hunk 13 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:108]: This is not WIP

File: code/datums/uplink/armor and tactical wear.dm (hunk 12 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:102]: This was a mistake. Will raise specialist armor types aside from riot to 4

File: code/datums/uplink/armor and tactical wear.dm (hunk 11 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:96]: Ignore what I said on ballistic, raising to 6 on both.

File: code/datums/uplink/armor and tactical wear.dm (hunk 10 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:90]: Raised to three, this is the affordable armor set.

File: code/datums/uplink/armor and tactical wear.dm (hunk 9 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:84]: Will raise to 2 TC.

File: code/datums/uplink/armor and tactical wear.dm (hunk 8 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:78]: This is not WIP.

File: code/datums/uplink/armor and tactical wear.dm (hunk 6 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:72]: This was raised to 4 TC.

File: code/datums/uplink/armor and tactical wear.dm (hunk 2 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:40]: Done.

File: code/datums/uplink/armor and tactical wear.dm (hunk 1 above)
Comment (Sparrowbird611) [reply code/datums/uplink/armor and tactical wear.dm:115]: Removed.

**Comment by Sparrowbird611** [comment]
Tweaks to prices, and the addition of three eye to the chemical section were made. I'm not really sure how to fix this conflict with Higgin's PR though.

**Comment by BurgerLUA** [comment]
conflicts and travis isn't working

**Comment by Sparrowbird611** [comment]
Closing, will upload a new version.

**Pull request closed without merging**